src/Microsoft.Azure.WebJobs.Host/Bindings/BindingFactory.cs

```diff
@@ -47,6 +47,19 @@ public IConverterManager ConverterManager
         }
 
         /// <summary>
+        /// Creating a type filter predicate around another rule. Filter doubles as a validator if it throws.  
+        /// </summary>
+        /// <param name="predicate">type predicate. Only apply inner rule if this predicate as applied to the user parameter type is true. </param>
+        /// <param name="innerRule">Inner rule. </param>
+        /// <returns></returns>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic")]
```

Comment:
My assumption is that you have no intention of fixing this. If you do, then make it static in this PR. Otherwise this is just code litter. That's why we generally shove all these nitpicky errors to the supression file. For things you plan on fixing fine, but generally all the violations are things we have no intention of ever fixing, so we don't need the code litter.